### ISSUE TITLE
[Question] Simulator - What is recall_repetitions, forget_repetitions, new_repetitions, and period_len

### ISSUE THREAD

**Issue by kuroahna** [open-spaced-repetition/fsrs4anki#16]
I'm testing the simulator and this is really cool, but I'm not 100% sure what `recall_repetitions`, `forget_repetitions`, `new_repetitions`, and `period_len` are

1. Is `recall_repetitions` how many repetitions it takes to pass a "review" card? For example, in Anki, when I review an existing card, and I press good, that means it is 1 repetition. Not sure what happens when you press Again (fail card). I suppose this is counted for in `forget_repetitions`
2. Is `forget_repetitions` how many repetitions it takes to review a card? For example, in Anki, in the options, there is "Lapses" section and I have the steps configured as "2 60", so would `forget_repetitions` be 2 here?
3. Is `new_repetitions` how many repetitions it takes to learn a "new" card? For example, in Anki, in the options, there is the "New cards" section, and I have the steps configured as "1 5 60", so would `new_repetitions` be 3 here?
4. Not sure what `period_len` is. In the graphs, I see "review cards per day (14 days average)" and "repetitions per day (14 days average)" since `period_len=14` by default. Does this mean that you are adding a data point for every 14 days? So if I want to see the "review cards per day" and "repetitions per day" for _every_ day, then I would set this to 1?

**Comment by L-M-Sherlock** [comment]
1. Yes, if you press hard, good and easy when you review, Anki will pass the card. So you only have 1 repetition for a card.
2. You can refer to the Anki simulator add-on https://ankiweb.net/shared/info/817108664
3. Same to above 
4. It’s used to smooth the curve. You can see this wiki page https://en.m.wikipedia.org/wiki/Moving_average

**Comment by kuroahna** [comment]
Thanks. I had a look at Anki simulator addon

![image](https://user-images.githubusercontent.com/85209455/191799274-0234f75e-7f04-4dc6-bad9-bc1ff7d5a4d0.png)

2. I assume `forget_repetitions` is the number of "Lapse steps" seen above
3. And same for `new_repetitions`. This is the number of "Learning steps"

**Comment by L-M-Sherlock** [comment]
Not very accurate. If you lapse steps is "2 60", you could press the 'again' button consistently. And press 'easy' could pass these steps.

**Comment by kuroahna** [comment]
Right, that makes sense.

Sorry, but I feel very dumb, I still don't really understand or know what `forget_repetitions` and `new_repetitions` is. I looked at Anki Simulator addon that you linked, but they don't mention anything about `forget_repetitions` or `new_repetitions`. I thought it was the number of lapse steps and number of learning steps, but from your response above, I don't think that's the case.

I also looked at their [source code](https://github.com/giovannihenriksen/Anki-Simulator/blob/master/src/anki_simulator/collection_simulator.py#L86-L87) and all I can find is

```python
        number_of_learning_steps: int,
        number_of_lapse_steps: int,
```

I don't see anything related to `forget_repetitions` or `new_repetitions` like you have mentioned. Should I leave things as default then?
```python
recall_repetitions = 1
forget_repetitions = 2
new_repetitions = 1
```

I wanted a rough estimation of how many reviews I would get using fsrs4anki using your simulator if I had lapse steps `2 60` and learning steps `1 5 60` in Anki, which is why I set them to

```python
recall_repetitions = 1
forget_repetitions = 2
new_repetitions = 3
```

but I'm still not sure if this is what I should do
